core/lib/dal/doc/FriProverDal.md

```diff
@@ -0,0 +1,26 @@
+# FriProverDal
+
+## Table Name
+
+prover_jobs_fri
+
+
+## `status` Diagram
+
+```mermaid
+---
+title: Status Diagram
+---
+stateDiagram-v2
+[*] --> queued : insert_prover_job
+queued --> in_progress : get_next_job
+in_progress --> successful : save_proof
+successful --> [*] 
+in_progress --> failed : save_proof_error
+failed --> [*]
+failed --> queued : requeue_stuck_jobs
+
+successful --> sent_to_server : save_successful_sent_proof
```

Comment:
`sent_to_server` should be removed, it's not achiavable

Reply:
@perekopskiy Hi, I've updated some of feedbacks. Please have a view